### PR TITLE
Medical GUI - Add setting to show Damage

### PR DESCRIPTION
**Pull request by LinkIsGrim**
**When merged this pull request will:**
- Add Setting & Injury List entry for displaying qualitative % of Damage Threshold taken on a given body part.
- Change hardcoded pain tooltip values in updateInjuryList to painUnconsciousThreshold variable.
- Take Damage Threshold into account when displaying body part damage color.

Screenshot:

### CODE REVIEW

**Comment by BrettMayson** [comment]
Setting for this? I personally would not want it

File: addons/medical_gui/initSettings.sqf [hunk 1]
```diff
@@ -51,3 +51,12 @@
     [[0, 1, 2], [ELSTRING(common,Disabled), ELSTRING(Medical_Treatment,Anyone), ELSTRING(Medical_Treatment,Medics)], 1],
     false
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(showDamageToolTip),
```
Comment (BrettMayson) [inline addons/medical_gui/initSettings.sqf:56]: Also nickpicky, but this isn't really a tooltip is it? Should it be entry instead?

**Review by BrettMayson — CHANGES_REQUESTED** [review]

File: addons/medical_gui/initSettings.sqf (hunk 1 above)
Comment (Salluci) [reply addons/medical_gui/initSettings.sqf:56]: sure, why the hell not

**Review by BrettMayson — APPROVED** [review]
Seems fine to me, fractures do seem to send anything to 100% damage.

File: addons/medical_gui/functions/fnc_updateInjuryList.sqf [hunk 2]
```diff
@@ -31,7 +31,26 @@ private _bodyPartName = [
...
+switch (true) do {
+    case (_selectionN > 3): {
+        _damageThreshold = LIMPING_DAMAGE_THRESHOLD * 4;
+    };
+    case (_selectionN > 1): {
+        _damageThreshold = FRACTURE_DAMAGE_THRESHOLD * 4;
+    };
+    case (_selectionN == 0): {
+        _damageThreshold = _damageThreshold / 2;
+    };
```
Comment (veteran29) [inline addons/medical_gui/functions/fnc_updateInjuryList.sqf]: IMO each `case` could use a comment to which body parts it applies, easier to follow.

File: addons/medical_gui/functions/fnc_updateInjuryList.sqf [hunk 3]
```diff
@@ -31,8 +31,47 @@ private _bodyPartName = [
     LSTRING(RightLeg)
 ] select _selectionN;
 
+
+
```
Comment (BrettMayson) [inline addons/medical_gui/functions/fnc_updateInjuryList.sqf:35]: ?

**Review by BrettMayson — APPROVED** [review]

File: addons/medical_gui/functions/fnc_updateInjuryList.sqf (hunk 3 above)
Comment (LinkIsGrim) [reply addons/medical_gui/functions/fnc_updateInjuryList.sqf:35]: Typos?

**Comment by Rythoka** [comment]
Just an opinion, but the terminology used here isn't great and I don't think it would be immediately obvious to most people what the difference between "major" and "severe" is. Also, "chronic" has nothing to do with severity, it just means that a conditions is persistent or long-lasting. "Chronic trauma" would be something closer to Carpal Tunnel Syndrome than a gunshot wound.

IMO a better set of categories would be "Minor," "Moderate," "Severe," and "Critical."

**Pull request merged**